jekyll/_data/sidenav.yml

```diff
@@ -308,9 +308,9 @@ en:
       link: insights-glossary
     - name: Insights snapshot badge
       link: insights-snapshot-badge
-    - name: Test Insights
+    - name: Test insights
```

Comment:
I think we still capitalize Insights, unless there's been new direction. It's the only feature we capitalize since it's different than insights in other development contexts.